Assets/Plugins/Source/Editor/ConfigEditors/ConfigEditor.cs

```diff
@@ -182,6 +182,10 @@ public void Load()
         public void Save(bool prettyPrint = true)
         {
             config.Write(prettyPrint);
+#if UNITY_EDITOR
+            AssetDatabase.SaveAssets();
+            AssetDatabase.Refresh();
+#endif
```

Comment:
Minor: Unnecessary #if check, this is all Editor code
```suggestion
            AssetDatabase.SaveAssets();
            AssetDatabase.Refresh();
```

Reply:
Removed #if check